### DISCUSSION TITLE
syncing LittleFS to flash

### DISCUSSION THREAD

**Discussion by myklemykle** [earlephilhower/arduino-pico#1599] [General]
Hi,

I may be mistaken, but in my situation it looks like LittleFS writes aren't sync'd to Flash until `LIttleFS.end()` is called.  Is that right?

Is there a preferred method to force a sync, other than restarting LittleFS with`LittleFS.end();LittleFS.begin(); ` ?

Thanks!

**Comment by earlephilhower** [comment]
There's no page cache like on a real OS, so all you need to do is `myfile.flush()` or close and re-open the file to guarantee it's on flash.  No need to stop/start the whole filesystem.
